doc/source/user_guide/adaptive_templates.rst

```diff
@@ -428,6 +428,39 @@ the function ``draw_objects_debug()`` can be used to plot the regions:
 
     Plot of rotor pocket regions drawn using the ``draw_objects()`` function.
 
+Adding imported DXF geometries to adaptive templates
+****************************************************
+Custom geometry can be imported to Motor-CAD from a DXF file. For details on how to import a custom
+geometry from DXF file, please see the 'Custom Machine Geometries' tutorial supplied with Motor-CAD.
```

Comment:
```suggestion
Custom geometry can be imported to Motor-CAD from a DXF file. For information on how to import a custom
geometry from a DXF file, see the 'Custom Machine Geometries' tutorial supplied with Motor-CAD.
```
Make the tutorial title a link to the URL for this tutorial? If not, replace single backticks with double quotation marks (") because surrounding in single backticks formats as italics.